src/data/dailylimits.txt

```diff
@@ -175,59 +175,59 @@ Cast	deliver your thesis!	_thesisDelivered
 Cast	lecture on mass	_pocketProfessorLectures	[fam(Pocket Professor)*ceil(sqrt(W))+(equipped(Pocket Professor Memory Chip)*2)]
 Cast	lecture on relativity	_pocketProfessorLectures	[fam(Pocket Professor)*ceil(sqrt(W))+(equipped(Pocket Professor Memory Chip)*2)]
 Cast	lecture on velocity	_pocketProfessorLectures	[fam(Pocket Professor)*ceil(sqrt(W))+(equipped(Pocket Professor Memory Chip)*2)]
-Chew	mansquito serum	_mansquitoSerumUsed
 Chew	The Author's ink	_authorsInkUsed
 Chew	The Inquisitor's unidentifiable object	_inquisitorsUnidentifiableObjectUsed
+Chew	mansquito serum	_mansquitoSerumUsed
 Chew	turkey blaster	_turkeyBlastersUsed	3
-Drink	Doc Clock's thyme cocktail	_docClocksThymeCocktailDrunk
-Drink	The Mad Liquor	_madLiquorDrunk
-Drink	Miss Graves' vermouth	_missGravesVermouthDrunk
-Drink	glass of "milk"	_speakeasyDrinksDrunk	3
-Drink	Cup of "tea"	_speakeasyDrinksDrunk	3
-Drink	thermos of "whiskey"	_speakeasyDrinksDrunk	3
-Drink	Lucky Lindy	_speakeasyDrinksDrunk	3
 Drink	Bee's Knees	_speakeasyDrinksDrunk	3
-Drink	Sockdollager	_speakeasyDrinksDrunk	3
-Drink	Ish Kabibble	_speakeasyDrinksDrunk	3
+Drink	Cup of "tea"	_speakeasyDrinksDrunk	3
```

Comment:
if you're trying to match the item names, I think this should be lowercase

Reply:
copy and pasting from the wiki probably not always smart for this ok

Reply:
and re-sort, as you've got case-sensitive sorting ;)